bindings/python/tests/fdbcli_tests.py

```diff
@@ -636,6 +636,63 @@ def tenants(logger):
     assert('printable' in json_output['tenant']['prefix'])
     assert(json_output['tenant']['tenant_state'] == 'ready')
 
+    output = run_fdbcli_command('gettenant tenant2')
+    lines = output.split('\n')
+    assert len(lines) == 4
+    assert lines[0].strip().startswith('id: ')
+    assert lines[1].strip().startswith('prefix: ')
+    assert lines[2].strip() == 'tenant state: ready'
+    assert lines[3].strip() == 'tenant group: tenant_group2'
+
+    output = run_fdbcli_command('gettenant tenant2 JSON')
+    json_output = json.loads(output, strict=False)
+    assert(len(json_output) == 2)
+    assert('tenant' in json_output)
+    assert(json_output['type'] == 'success')
+    assert(len(json_output['tenant']) == 4)
+    assert('id' in json_output['tenant'])
+    assert('prefix' in json_output['tenant'])
+    assert(json_output['tenant']['tenant_state'] == 'ready')
+    assert('tenant_group' in json_output['tenant'])
+    assert(len(json_output['tenant']['tenant_group']) == 2)
+    assert('base64' in json_output['tenant']['tenant_group'])
+    assert(json_output['tenant']['tenant_group']['printable'] == 'tenant_group2')
+
+    output = run_fdbcli_command('configuretenant tenant tenant_group=tenant_group1')
+    assert output == 'The configuration for tenant `tenant\' has been updated'
+
+    output = run_fdbcli_command('gettenant tenant')
+    lines = output.split('\n')
+    assert len(lines) == 4
+    assert lines[3].strip() == 'tenant group: tenant_group1'
+
+    output = run_fdbcli_command('configuretenant tenant tenant_group=tenant_group1 tenant_group=tenant_group2')
```

Comment:
I am not sure I understand this correctly, but it seems that this command will add the tenant only to `tenant_group2`. This seems a bit counter-intuitive, shouldn't this instead return with an error? Or is there some motivation I am missing

Reply:
That's a fair question. The way I structured the configure command, it takes an arbitrary sequence of configuration parameters and the last one of each name wins. There really isn't a good reason for that, so I can change it to disallow setting the same parameter twice.